Dart/src/com/jetbrains/lang/dart/flutter/FlutterUtil.java

```diff
@@ -0,0 +1,47 @@
+package com.jetbrains.lang.dart.flutter;
+
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.module.ModuleType;
+import com.jetbrains.lang.dart.sdk.DartSdk;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.File;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+public class FlutterUtil {
+
+  private static final String FLUTTER_MODULE_TYPE_NAME = "Flutter";
+
+  /**
+   * Test if the given module is a Flutter module.
+   *
+   * @param module the module to test
+   * @return true if the module is a Flutter module, false otherwise
+   */
+  public static boolean isFlutterModule(@NotNull Module module) {
+    final ModuleType moduleType = ModuleType.get(module);
+    return FLUTTER_MODULE_TYPE_NAME.equals(moduleType.getName());
```

Comment:
I think it's better to check module type id. Module type name can be safely changed (for example someone may decide to rename 'Flutter' to 'Flutter Module' as all other module types have 'Module' word), but module type id is less likely to be ever changed as it is a breaking change.